src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -31926,10 +31822,8 @@ void M()
             comp.VerifyDiagnostics(
                 // (13,17): warning CS8632: The annotation for nullable reference types should only be used in code within a '[NonNullTypes(true)]' context.
                 //         object? _f;
-                Diagnostic(ErrorCode.WRN_MissingNonNullTypesContextForAnnotation, "_f").WithLocation(13, 17),
-                // (16,22): warning CS8602: Possible dereference of a null reference.
-                //             _value = _f.GetHashCode();
-                Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "_f").WithLocation(16, 22));
+                Diagnostic(ErrorCode.WRN_MissingNonNullTypesContextForAnnotation, "_f").WithLocation(13, 17)
+                );
```

Comment:
This test should use `NullableWalker` since that was where the failure was.

Reply:
Good catch. Thanks

---
In reply to: [208400657](https://github.com/dotnet/roslyn/pull/29147#discussion_r208400657) [](ancestors = 208400657)